src/openrct2/park/ParkFile.cpp

```diff
@@ -162,6 +162,10 @@ namespace OpenRCT2
             {
                 UpdateTrackElementsRideType();
             }
+            if (os.GetHeader().TargetVersion < 17)
+            {
+                UpdateTrackBrakeSpeed();
```

Comment:
This should be moved to somewhere after RTD objects would be loaded, if it is not already there. Likewise with S6 and S4z